[libb2] generates arm64 binaries for triplet x64-osx

**Host Environment**
- OS: macOS with apple m1
- Compiler: clang

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install libb2`

**Failure logs** 
```
➜  vcpkg git:(qt6-base) ✗ ./vcpkg install --triplet=x64-osx --host-triplet=x64-osx --binarysource=clear
Detecting compiler hash for triplet x64-osx...
The following packages will be built and installed:
    libb2[core]:x64-osx -> 0.98.1#4
Starting package 1/1: libb2:x64-osx
Building package libb2[core]:x64-osx...
-- Using cached /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/BLAKE2-libb2-2c5142f12a2cd52f3ee0a43e50a3a76f75badf85.tar.gz
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/BLAKE2-libb2-2c5142f12a2cd52f3ee0a43e50a3a76f75badf85.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- DEBUG: --host=x86_64-apple-darwin
-- Generating configure for x64-osx
-- Finished generating configure for x64-osx
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
-- Installing x64-osx-dbg
-- Building x64-osx-rel
-- Installing x64-osx-rel
-- Fixing pkgconfig file: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libb2_x64-osx/lib/pkgconfig/libb2.pc
-- Fixing pkgconfig file: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libb2_x64-osx/debug/lib/pkgconfig/libb2.pc
-- Installing: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libb2_x64-osx/share/libb2/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package libb2[core]:x64-osx... done
Installing package libb2[core]:x64-osx...
Installing package libb2[core]:x64-osx... done
Elapsed time for package libb2:x64-osx: 14.82 s

Total elapsed time: 14.82 s

➜  vcpkg git:(qt6-base) ✗ lipo -archs ../vcpkg_installed/x64-osx/lib/libb2.a
arm64

```

<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/config-x64-osx-dbg-out.log</summary>

```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for x86_64-apple-darwin-strip... no
checking for strip... strip
checking for a race-free mkdir -p... ./../src/6f75badf85-b3a5d32c8d.clean/install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for x86_64-apple-darwin-gcc... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for explicit_bzero... no
checking for explicit_memset... no
checking for memset_s... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for stddef.h... yes
checking for stdint.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for gcc option to support OpenMP... unsupported
checking whether byte ordering is bigendian... no
checking for inline... inline
checking for size_t... yes
checking for uint32_t... yes
checking for uint64_t... yes
checking for uint8_t... yes
checking for memset... yes
checking build system type... aarch64-apple-darwin20.4.0
checking host system type... x86_64-apple-darwin
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/local/bin/gsed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... no
checking for x86_64-apple-darwin-dumpbin... no
checking for x86_64-apple-darwin-link... no
checking for dumpbin... no
checking for link... link -dump
checking the name lister (nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 786432
checking how to convert aarch64-apple-darwin20.4.0 file names to x86_64-apple-darwin format... func_convert_file_noop
checking how to convert aarch64-apple-darwin20.4.0 file names to toolchain format... func_convert_file_noop
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files... -r
checking for x86_64-apple-darwin-objdump... no
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-apple-darwin-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-apple-darwin-ar... no
checking for ar... ar
checking for archiver @FILE support... no
checking for x86_64-apple-darwin-strip... strip
checking for x86_64-apple-darwin-ranlib... no
checking for ranlib... ranlib
checking command to parse nm output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for x86_64-apple-darwin-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for x86_64-apple-darwin-dsymutil... no
checking for dsymutil... dsymutil
checking for x86_64-apple-darwin-nmedit... no
checking for nmedit... nmedit
checking for x86_64-apple-darwin-lipo... no
checking for lipo... lipo
checking for x86_64-apple-darwin-otool... no
checking for otool... otool
checking for x86_64-apple-darwin-otool64... no
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... yes
checking for gcc option to produce PIC... -fno-common -DPIC
checking if gcc PIC flag -fno-common -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking whether C compiler accepts -O3... yes
checking whether C compiler accepts -march=native... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating libb2.pc
config.status: creating src/config.h
config.status: executing depfiles commands
config.status: executing libtool commands
```
<details>

It reports `checking for x86_64-apple-darwin-gcc... no`

Ah ok
<details><summary>config.log-x64-osx-rel.log</summary>

```
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by libb2 configure 0.98.1, which was
generated by GNU Autoconf 2.71.  Invocation command line was

  $ ./../src/6f75badf85-b3a5d32c8d.clean/configure --host=x86_64-apple-darwin --disable-silent-rules --verbose --disable-shared --enable-static --prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx '--bindir=${prefix}/tools/libb2/bin' '--sbindir=${prefix}/tools/libb2/sbin' '--mandir=${prefix}/share/libb2' '--docdir=${prefix}/share/libb2' '--datarootdir=${prefix}/share/libb2'

## --------- ##
## Platform. ##
## --------- ##

hostname = MacBook-Pro.local
uname -m = arm64
uname -r = 20.4.0
uname -s = Darwin
uname -v = Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:41 PDT 2021; root:xnu-7195.101.2~1/RELEASE_ARM64_T8101

/usr/bin/uname -p = arm
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = Mach kernel version:
	 Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:41 PDT 2021; root:xnu-7195.101.2~1/RELEASE_ARM64_T8101
Kernel configured for up to 8 processors.
8 processors are physically available.
8 processors are logically available.
Processor type: arm64e (ARM64E)
Processors active: 0 1 2 3 4 5 6 7
Primary memory available: 16.00 gigabytes
Default processor set: 595 tasks, 3714 threads, 8 processors
Load average: 6.40, Mach factor: 2.14
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /usr/local/opt/node@14/bin/
PATH: /Users/leanderSchulten/anaconda3/bin/
PATH: /usr/local/bin/
PATH: /usr/bin/
PATH: /bin/
PATH: /usr/sbin/
PATH: /sbin/
PATH: /Library/TeX/texbin/
PATH: /usr/local/share/dotnet/
PATH: /opt/X11/bin/
PATH: /Library/Apple/usr/bin/
PATH: /Library/Frameworks/Mono.framework/Versions/Current/Commands/
PATH: /Applications/Wireshark.app/Contents/MacOS/
PATH: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/ninja-1.10.1-osx/


## ----------- ##
## Core tests. ##
## ----------- ##

configure:2529: looking for aux files: config.guess config.sub ltmain.sh compile missing install-sh
configure:2542:  trying ./../src/6f75badf85-b3a5d32c8d.clean/
configure:2571:   ./../src/6f75badf85-b3a5d32c8d.clean/config.guess found
configure:2571:   ./../src/6f75badf85-b3a5d32c8d.clean/config.sub found
configure:2571:   ./../src/6f75badf85-b3a5d32c8d.clean/ltmain.sh found
configure:2571:   ./../src/6f75badf85-b3a5d32c8d.clean/compile found
configure:2571:   ./../src/6f75badf85-b3a5d32c8d.clean/missing found
configure:2553:   ./../src/6f75badf85-b3a5d32c8d.clean/install-sh found
configure:2703: checking for a BSD-compatible install
configure:2776: result: /usr/bin/install -c
configure:2787: checking whether build environment is sane
configure:2842: result: yes
configure:2898: checking for x86_64-apple-darwin-strip
configure:2933: result: no
configure:2943: checking for strip
configure:2964: found /usr/bin/strip
configure:2975: result: strip
configure:3001: checking for a race-free mkdir -p
configure:3045: result: ./../src/6f75badf85-b3a5d32c8d.clean/install-sh -c -d
configure:3052: checking for gawk
configure:3087: result: no
configure:3052: checking for mawk
configure:3087: result: no
configure:3052: checking for nawk
configure:3087: result: no
configure:3052: checking for awk
configure:3073: found /usr/bin/awk
configure:3084: result: awk
configure:3095: checking whether make sets $(MAKE)
configure:3118: result: yes
configure:3148: checking whether make supports nested variables
configure:3166: result: yes
configure:3316: checking for x86_64-apple-darwin-gcc
configure:3351: result: no
configure:3361: checking for gcc
configure:3382: found /usr/bin/gcc
configure:3393: result: gcc
configure:3746: checking for C compiler version
configure:3755: gcc --version >&5
Apple clang version 12.0.5 (clang-1205.0.22.9)
Target: arm64-apple-darwin20.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
configure:3766: $? = 0
configure:3755: gcc -v >&5
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 12.0.5 (clang-1205.0.22.9)
Target: arm64-apple-darwin20.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:3766: $? = 0
configure:3755: gcc -V >&5
clang: error: argument to '-V' is missing (expected 1 value)
clang: error: no input files
configure:3766: $? = 1
configure:3755: gcc -qversion >&5
clang: error: unknown argument '-qversion'; did you mean '--version'?
clang: error: no input files
configure:3766: $? = 1
configure:3755: gcc -version >&5
clang: error: unknown argument '-version'; did you mean '--version'?
clang: error: no input files
configure:3766: $? = 1
configure:3786: checking whether the C compiler works
configure:3808: gcc -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
configure:3812: $? = 0
configure:3862: result: yes
configure:3865: checking for C compiler default output file name
configure:3867: result: a.out
configure:3873: checking for suffix of executables
configure:3880: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
configure:3884: $? = 0
configure:3907: result: 
configure:3929: checking whether we are cross compiling
configure:3937: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
configure:3941: $? = 0
configure:3948: ./conftest
configure:3952: $? = 0
configure:3967: result: no
configure:3972: checking for suffix of object files
configure:3995: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:3999: $? = 0
configure:4021: result: o
configure:4025: checking whether the compiler supports GNU C
configure:4045: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4045: $? = 0
configure:4055: result: yes
configure:4066: checking whether gcc accepts -g
configure:4087: gcc -c -g -DNDEBUG conftest.c >&5
configure:4087: $? = 0
configure:4131: result: yes
configure:4151: checking for gcc option to enable C11 features
configure:4166: gcc  -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4166: $? = 0
configure:4184: result: none needed
configure:4300: checking whether gcc understands -c and -o together
configure:4323: gcc -c conftest.c -o conftest2.o
configure:4326: $? = 0
configure:4323: gcc -c conftest.c -o conftest2.o
configure:4326: $? = 0
configure:4338: result: yes
configure:4358: checking whether make supports the include directive
configure:4373: make -f confmf.GNU && cat confinc.out
this is the am__doit target
configure:4376: $? = 0
configure:4395: result: yes (GNU style)
configure:4421: checking dependency style of gcc
configure:4533: result: gcc3
configure:4551: checking for explicit_bzero
configure:4551: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
Undefined symbols for architecture x86_64:
  "_explicit_bzero", referenced from:
      _main in conftest-a755fc.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
configure:4551: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| /* end confdefs.h.  */
| /* Define explicit_bzero to an innocuous variant, in case <limits.h> declares explicit_bzero.
|    For example, HP-UX 11i <limits.h> declares gettimeofday.  */
| #define explicit_bzero innocuous_explicit_bzero
| 
| /* System header to define __stub macros and hopefully few prototypes,
|    which can conflict with char explicit_bzero (); below.  */
| 
| #include <limits.h>
| #undef explicit_bzero
| 
| /* Override any GCC internal prototype to avoid an error.
|    Use char because int might match the return type of a GCC
|    builtin and then its argument prototype would still apply.  */
| #ifdef __cplusplus
| extern "C"
| #endif
| char explicit_bzero ();
| /* The GNU C library defines this for functions which it implements
|     to always fail with ENOSYS.  Some functions are actually named
|     something starting with __ and the normal name is an alias.  */
| #if defined __stub_explicit_bzero || defined __stub___explicit_bzero
| choke me
| #endif
| 
| int
| main (void)
| {
| return explicit_bzero ();
|   ;
|   return 0;
| }
configure:4551: result: no
configure:4558: checking for explicit_memset
configure:4558: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
Undefined symbols for architecture x86_64:
  "_explicit_memset", referenced from:
      _main in conftest-56b664.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
configure:4558: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| /* end confdefs.h.  */
| /* Define explicit_memset to an innocuous variant, in case <limits.h> declares explicit_memset.
|    For example, HP-UX 11i <limits.h> declares gettimeofday.  */
| #define explicit_memset innocuous_explicit_memset
| 
| /* System header to define __stub macros and hopefully few prototypes,
|    which can conflict with char explicit_memset (); below.  */
| 
| #include <limits.h>
| #undef explicit_memset
| 
| /* Override any GCC internal prototype to avoid an error.
|    Use char because int might match the return type of a GCC
|    builtin and then its argument prototype would still apply.  */
| #ifdef __cplusplus
| extern "C"
| #endif
| char explicit_memset ();
| /* The GNU C library defines this for functions which it implements
|     to always fail with ENOSYS.  Some functions are actually named
|     something starting with __ and the normal name is an alias.  */
| #if defined __stub_explicit_memset || defined __stub___explicit_memset
| choke me
| #endif
| 
| int
| main (void)
| {
| return explicit_memset ();
|   ;
|   return 0;
| }
configure:4558: result: no
configure:4565: checking for memset_s
configure:4565: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
configure:4565: $? = 0
configure:4565: result: yes
configure:4576: checking for stdio.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for stdlib.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for string.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for inttypes.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for stdint.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for strings.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for sys/stat.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for sys/types.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4576: checking for unistd.h
configure:4576: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4576: $? = 0
configure:4576: result: yes
configure:4601: checking for stddef.h
configure:4601: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4601: $? = 0
configure:4601: result: yes
configure:4607: checking for stdint.h
configure:4607: result: yes
configure:4613: checking for stdlib.h
configure:4613: result: yes
configure:4619: checking for string.h
configure:4619: result: yes
configure:4637: checking for gcc option to support OpenMP
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3  -DNDEBUG conftest.c >&5
conftest.c:28:2: error: "OpenMP not supported"
#error "OpenMP not supported"
 ^
conftest.c:30:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
2 errors generated.
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -fopenmp -DNDEBUG conftest.c >&5
clang: error: unsupported option '-fopenmp'
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -xopenmp -DNDEBUG conftest.c >&5
clang: error: language not recognized: 'openmp'
clang: warning: conftest.c: 'linker' input unused [-Wunused-command-line-argument]
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -openmp -DNDEBUG conftest.c >&5
conftest.c:28:2: error: "OpenMP not supported"
#error "OpenMP not supported"
 ^
conftest.c:30:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
2 errors generated.
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -mp -DNDEBUG conftest.c >&5
clang: error: unknown argument: '-mp'
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -omp -DNDEBUG conftest.c >&5
conftest.c:28:2: error: "OpenMP not supported"
#error "OpenMP not supported"
 ^
conftest.c:30:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
2 errors generated.
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -qsmp=omp -DNDEBUG conftest.c >&5
clang: error: unknown argument: '-qsmp=omp'
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -homp -DNDEBUG conftest.c >&5
clang: error: unknown argument: '-homp'
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -Popenmp -DNDEBUG conftest.c >&5
clang: error: unknown argument '-Popenmp'; did you mean '-fopenmp'?
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4659: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 --openmp -DNDEBUG conftest.c >&5
clang: error: unsupported option '--openmp'; did you mean '-fopenmp'?
configure:4659: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| 
| #ifndef _OPENMP
| #error "OpenMP not supported"
| #endif
| #include <omp.h>
| int main (void) { return omp_get_num_threads (); }
| 
configure:4694: result: unsupported
configure:4704: checking whether byte ordering is bigendian
configure:4720: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4720: $? = 0
configure:4766: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4766: $? = 0
configure:4785: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
conftest.c:33:4: error: use of undeclared identifier 'not'
                 not big endian
                 ^
1 error generated.
configure:4785: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| /* end confdefs.h.  */
| #include <sys/types.h>
| 		#include <sys/param.h>
| 
| int
| main (void)
| {
| #if BYTE_ORDER != BIG_ENDIAN
| 		 not big endian
| 		#endif
| 
|   ;
|   return 0;
| }
configure:4919: result: no
configure:4939: checking for inline
configure:4956: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4956: $? = 0
configure:4965: result: inline
configure:4983: checking for size_t
configure:4983: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4983: $? = 0
configure:4983: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
conftest.c:58:21: error: expected expression
if (sizeof ((size_t)))
                    ^
1 error generated.
configure:4983: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define NATIVE_LITTLE_ENDIAN 1
| /* end confdefs.h.  */
| #include <stddef.h>
| #ifdef HAVE_STDIO_H
| # include <stdio.h>
| #endif
| #ifdef HAVE_STDLIB_H
| # include <stdlib.h>
| #endif
| #ifdef HAVE_STRING_H
| # include <string.h>
| #endif
| #ifdef HAVE_INTTYPES_H
| # include <inttypes.h>
| #endif
| #ifdef HAVE_STDINT_H
| # include <stdint.h>
| #endif
| #ifdef HAVE_STRINGS_H
| # include <strings.h>
| #endif
| #ifdef HAVE_SYS_TYPES_H
| # include <sys/types.h>
| #endif
| #ifdef HAVE_SYS_STAT_H
| # include <sys/stat.h>
| #endif
| #ifdef HAVE_UNISTD_H
| # include <unistd.h>
| #endif
| int
| main (void)
| {
| if (sizeof ((size_t)))
| 	    return 0;
|   ;
|   return 0;
| }
configure:4983: result: yes
configure:4993: checking for uint32_t
configure:4993: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:4993: $? = 0
configure:4993: result: yes
configure:5005: checking for uint64_t
configure:5005: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:5005: $? = 0
configure:5005: result: yes
configure:5017: checking for uint8_t
configure:5017: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:5017: $? = 0
configure:5017: result: yes
configure:5031: checking for memset
configure:5031: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c  >&5
conftest.c:43:6: warning: incompatible redeclaration of library function 'memset' [-Wincompatible-library-redeclaration]
char memset ();
     ^
conftest.c:43:6: note: 'memset' is a builtin with type 'void *(void *, int, unsigned long)'
1 warning generated.
configure:5031: $? = 0
configure:5031: result: yes
configure:5071: checking build system type
configure:5086: result: aarch64-apple-darwin20.4.0
configure:5106: checking host system type
configure:5120: result: x86_64-apple-darwin
configure:5161: checking how to print strings
configure:5188: result: printf
configure:5209: checking for a sed that does not truncate output
configure:5279: result: /usr/local/bin/gsed
configure:5297: checking for grep that handles long lines and -e
configure:5361: result: /usr/bin/grep
configure:5366: checking for egrep
configure:5434: result: /usr/bin/grep -E
configure:5439: checking for fgrep
configure:5507: result: /usr/bin/grep -F
configure:5543: checking for ld used by gcc
configure:5611: result: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
configure:5618: checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld
configure:5634: result: no
configure:5646: checking for BSD- or MS-compatible name lister (nm)
configure:5701: result: no
configure:5715: checking for x86_64-apple-darwin-dumpbin
configure:5750: result: no
configure:5715: checking for x86_64-apple-darwin-link
configure:5750: result: no
configure:5764: checking for dumpbin
configure:5799: result: no
configure:5764: checking for link
configure:5785: found /bin/link
configure:5796: result: link -dump
configure:5841: checking the name lister (nm) interface
configure:5849: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:5852: nm "conftest.o"
configure:5855: output
0000000000000000 S _some_variable
configure:5862: result: BSD nm
configure:5865: checking whether ln -s works
configure:5869: result: yes
configure:5877: checking the maximum length of command line arguments
configure:6009: result: 786432
configure:6057: checking how to convert aarch64-apple-darwin20.4.0 file names to x86_64-apple-darwin format
configure:6098: result: func_convert_file_noop
configure:6105: checking how to convert aarch64-apple-darwin20.4.0 file names to toolchain format
configure:6126: result: func_convert_file_noop
configure:6133: checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files
configure:6141: result: -r
configure:6175: checking for x86_64-apple-darwin-objdump
configure:6210: result: no
configure:6220: checking for objdump
configure:6241: found /usr/bin/objdump
configure:6252: result: objdump
configure:6284: checking how to recognize dependent libraries
configure:6485: result: pass_all
configure:6530: checking for x86_64-apple-darwin-dlltool
configure:6565: result: no
configure:6575: checking for dlltool
configure:6610: result: no
configure:6640: checking how to associate runtime and link libraries
configure:6668: result: printf %s\n
configure:6684: checking for x86_64-apple-darwin-ar
configure:6719: result: no
configure:6733: checking for ar
configure:6754: found /usr/bin/ar
configure:6765: result: ar
configure:6802: checking for archiver @FILE support
configure:6820: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:6820: $? = 0
configure:6824: ar cru libconftest.a @conftest.lst >&5
ar: @conftest.lst: No such file or directory
configure:6827: $? = 1
configure:6847: result: no
configure:6865: checking for x86_64-apple-darwin-strip
configure:6897: result: strip
configure:6974: checking for x86_64-apple-darwin-ranlib
configure:7009: result: no
configure:7019: checking for ranlib
configure:7040: found /usr/bin/ranlib
configure:7051: result: ranlib
configure:7153: checking command to parse nm output from gcc object
configure:7307: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:7310: $? = 0
configure:7314: nm conftest.o \| sed -n -e 's/^.*[ ]\([BCDEGRST][BCDEGRST]*\)[ ][ ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' | sed '/ __gnu_lto/d' \> conftest.nm
configure:7317: $? = 0
cannot find nm_test_var in conftest.nm
configure:7307: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:7310: $? = 0
configure:7314: nm conftest.o \| sed -n -e 's/^.*[ ]\([BCDEGRST][BCDEGRST]*\)[ ][ ]*_\([_A-Za-z][_A-Za-z0-9]*\)$/\1 _\2 \2/p' | sed '/ __gnu_lto/d' \> conftest.nm
configure:7317: $? = 0
configure:7383: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   conftest.c conftstm.o >&5
configure:7386: $? = 0
configure:7424: result: ok
configure:7471: checking for sysroot
configure:7502: result: no
configure:7509: checking for a working dd
configure:7553: result: /bin/dd
configure:7557: checking how to truncate binary pipes
configure:7573: result: /bin/dd bs=4096 count=1
configure:7865: checking for x86_64-apple-darwin-mt
configure:7900: result: no
configure:7910: checking for mt
configure:7945: result: no
configure:7965: checking if : is a manifest tool
configure:7972: : '-?'
configure:7980: result: no
configure:7996: checking for x86_64-apple-darwin-dsymutil
configure:8031: result: no
configure:8041: checking for dsymutil
configure:8062: found /usr/bin/dsymutil
configure:8073: result: dsymutil
configure:8098: checking for x86_64-apple-darwin-nmedit
configure:8133: result: no
configure:8143: checking for nmedit
configure:8164: found /usr/bin/nmedit
configure:8175: result: nmedit
configure:8200: checking for x86_64-apple-darwin-lipo
configure:8235: result: no
configure:8245: checking for lipo
configure:8266: found /usr/bin/lipo
configure:8277: result: lipo
configure:8302: checking for x86_64-apple-darwin-otool
configure:8337: result: no
configure:8347: checking for otool
configure:8368: found /usr/bin/otool
configure:8379: result: otool
configure:8404: checking for x86_64-apple-darwin-otool64
configure:8439: result: no
configure:8449: checking for otool64
configure:8484: result: no
configure:8529: checking for -single_module linker flag
gcc -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3   -o libconftest.dylib -dynamiclib -Wl,-single_module conftest.c
configure:8563: result: yes
configure:8566: checking for -exported_symbols_list linker flag
configure:8587: gcc -o conftest -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG   -Wl,-exported_symbols_list,conftest.sym conftest.c  >&5
configure:8587: $? = 0
configure:8598: result: yes
configure:8601: checking for -force_load linker flag
gcc -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -c -o conftest.o conftest.c
ar cru libconftest.a conftest.o
ranlib libconftest.a
gcc -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3   -o conftest conftest.c -Wl,-force_load,./libconftest.a
configure:8634: result: yes
configure:8701: checking for dlfcn.h
configure:8701: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG conftest.c >&5
configure:8701: $? = 0
configure:8701: result: yes
configure:8960: checking for objdir
configure:8976: result: .libs
configure:9240: checking if gcc supports -fno-rtti -fno-exceptions
configure:9259: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG -fno-rtti -fno-exceptions conftest.c >&5
configure:9263: $? = 0
configure:9276: result: yes
configure:9634: checking for gcc option to produce PIC
configure:9642: result: -fno-common -DPIC
configure:9650: checking if gcc PIC flag -fno-common -DPIC works
configure:9669: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG -fno-common -DPIC -DPIC conftest.c >&5
configure:9673: $? = 0
configure:9686: result: yes
configure:9715: checking if gcc static flag -static works
configure:9744: result: no
configure:9759: checking if gcc supports -c -o file.o
configure:9781: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3 -DNDEBUG -o out/conftest2.o conftest.c >&5
configure:9785: $? = 0
configure:9807: result: yes
configure:9815: checking if gcc supports -c -o file.o
configure:9863: result: yes
configure:9896: checking whether the gcc linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries
configure:11162: result: yes
configure:11403: checking dynamic linker characteristics
configure:12224: result: darwin dyld
configure:12346: checking how to hardcode library paths into programs
configure:12371: result: immediate
configure:12923: checking whether stripping libraries is possible
configure:12937: result: yes
configure:12963: checking if libtool supports shared libraries
configure:12965: result: yes
configure:12968: checking whether to build shared libraries
configure:12993: result: no
configure:12996: checking whether to build static libraries
configure:13000: result: yes
configure:13065: checking whether C compiler accepts -O3
configure:13085: gcc -c -fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3  -O3 -DNDEBUG conftest.c >&5
conftest.c:34:36: warning: result of comparison of constant -1000 with expression of type 'char' is always true [-Wtautological-constant-out-of-range-compare]
char x[42U], fodder = 0;if (fodder > -1000 && fgets(x,1000,stdin)) puts(x)
                            ~~~~~~ ^ ~~~~~
1 warning generated.
configure:13085: $? = 0
configure:13094: result: yes
configure:13104: checking whether C compiler accepts -march=native
configure:13124: gcc -c -O3  -march=native -DNDEBUG conftest.c >&5
clang: error: the clang compiler does not support '-march=native'
configure:13124: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libb2"
| #define PACKAGE_TARNAME "libb2"
| #define PACKAGE_VERSION "0.98.1"
| #define PACKAGE_STRING "libb2 0.98.1"
| #define PACKAGE_BUGREPORT "contact@blake2.net"
| #define PACKAGE_URL "https://blake2.net"
| #define PACKAGE "libb2"
| #define VERSION "0.98.1"
| #define HAVE_MEMSET_S 1
| #define HAVE_STDIO_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_UNISTD_H 1
| #define STDC_HEADERS 1
| #define HAVE_STDDEF_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define NATIVE_LITTLE_ENDIAN 1
| #define HAVE_MEMSET 1
| #define HAVE_DLFCN_H 1
| #define LT_OBJDIR ".libs/"
| /* end confdefs.h.  */
| #include <stdio.h>
| int
| main (void)
| {
| char x[42U], fodder = 0;if (fodder > -1000 && fgets(x,1000,stdin)) puts(x)
|   ;
|   return 0;
| }
configure:13133: result: no
configure:14212: checking that generated files are newer than configure
configure:14218: result: done
configure:14249: creating ./config.status

## ---------------------- ##
## Running config.status. ##
## ---------------------- ##

This file was extended by libb2 config.status 0.98.1, which was
generated by GNU Autoconf 2.71.  Invocation command line was

  CONFIG_FILES    = 
  CONFIG_HEADERS  = 
  CONFIG_LINKS    = 
  CONFIG_COMMANDS = 
  $ ./config.status 

on MacBook-Pro.local

config.status:1091: creating Makefile
config.status:1091: creating src/Makefile
config.status:1091: creating libb2.pc
config.status:1091: creating src/config.h
config.status:1305: executing depfiles commands
config.status:1382: cd src       && sed -e '/# am--include-marker/d' Makefile         | make -f - am--depfiles
config.status:1387: $? = 0
config.status:1305: executing libtool commands

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_build=aarch64-apple-darwin20.4.0
ac_cv_c_bigendian=no
ac_cv_c_compiler_gnu=yes
ac_cv_c_inline=inline
ac_cv_c_uint32_t=yes
ac_cv_c_uint64_t=yes
ac_cv_c_uint8_t=yes
ac_cv_env_CC_set=
ac_cv_env_CC_value=
ac_cv_env_CFLAGS_set=set
ac_cv_env_CFLAGS_value='-fPIC -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -O3'
ac_cv_env_CPPFLAGS_set=set
ac_cv_env_CPPFLAGS_value=-DNDEBUG
ac_cv_env_LDFLAGS_set=set
ac_cv_env_LDFLAGS_value=' '
ac_cv_env_LIBS_set=
ac_cv_env_LIBS_value=
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=set
ac_cv_env_host_alias_value=x86_64-apple-darwin
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_func_explicit_bzero=no
ac_cv_func_explicit_memset=no
ac_cv_func_memset=yes
ac_cv_func_memset_s=yes
ac_cv_header_dlfcn_h=yes
ac_cv_header_inttypes_h=yes
ac_cv_header_stddef_h=yes
ac_cv_header_stdint_h=yes
ac_cv_header_stdio_h=yes
ac_cv_header_stdlib_h=yes
ac_cv_header_string_h=yes
ac_cv_header_strings_h=yes
ac_cv_header_sys_stat_h=yes
ac_cv_header_sys_types_h=yes
ac_cv_header_unistd_h=yes
ac_cv_host=x86_64-apple-darwin
ac_cv_objext=o
ac_cv_path_EGREP='/usr/bin/grep -E'
ac_cv_path_FGREP='/usr/bin/grep -F'
ac_cv_path_GREP=/usr/bin/grep
ac_cv_path_SED=/usr/local/bin/gsed
ac_cv_path_install='/usr/bin/install -c'
ac_cv_path_lt_DD=/bin/dd
ac_cv_prog_AWK=awk
ac_cv_prog_STRIP=strip
ac_cv_prog_ac_ct_AR=ar
ac_cv_prog_ac_ct_CC=gcc
ac_cv_prog_ac_ct_DSYMUTIL=dsymutil
ac_cv_prog_ac_ct_DUMPBIN='link -dump'
ac_cv_prog_ac_ct_LIPO=lipo
ac_cv_prog_ac_ct_NMEDIT=nmedit
ac_cv_prog_ac_ct_OBJDUMP=objdump
ac_cv_prog_ac_ct_OTOOL=otool
ac_cv_prog_ac_ct_RANLIB=ranlib
ac_cv_prog_ac_ct_STRIP=strip
ac_cv_prog_c_openmp=unsupported
ac_cv_prog_cc_c11=
ac_cv_prog_cc_g=yes
ac_cv_prog_cc_stdc=
ac_cv_prog_make_make_set=yes
ac_cv_type_size_t=yes
am_cv_CC_dependencies_compiler_type=gcc3
am_cv_make_support_nested_variables=yes
am_cv_prog_cc_c_o=yes
ax_cv_check_cflags___O3=yes
ax_cv_check_cflags___march_native=no
lt_cv_apple_cc_single_mod=yes
lt_cv_ar_at_file=no
lt_cv_deplibs_check_method=pass_all
lt_cv_file_magic_cmd='$MAGIC_CMD'
lt_cv_file_magic_test_file=
lt_cv_ld_exported_symbols_list=yes
lt_cv_ld_force_load=yes
lt_cv_ld_reload_flag=-r
lt_cv_nm_interface='BSD nm'
lt_cv_objdir=.libs
lt_cv_path_LD=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
lt_cv_path_NM=no
lt_cv_path_mainfest_tool=no
lt_cv_prog_compiler_c_o=yes
lt_cv_prog_compiler_pic='-fno-common -DPIC'
lt_cv_prog_compiler_pic_works=yes
lt_cv_prog_compiler_rtti_exceptions=yes
lt_cv_prog_compiler_static_works=no
lt_cv_prog_gnu_ld=no
lt_cv_sharedlib_from_linklib_cmd='printf %s\n'
lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[	 ]\([BCDEGRST][BCDEGRST]*\)[	 ][	 ]*_\([_A-Za-z][_A-Za-z0-9]*\)$/\1 _\2 \2/p'\'' | sed '\''/ __gnu_lto/d'\'''
lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/  {"\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(lib.*\)$/  {"\1", (void *) \&\1},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/  {"lib\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/extern char \1;/p'\'''
lt_cv_sys_global_symbol_to_import=
lt_cv_sys_max_cmd_len=786432
lt_cv_to_host_file_cmd=func_convert_file_noop
lt_cv_to_tool_file_cmd=func_convert_file_noop
lt_cv_truncate_bin='/bin/dd bs=4096 count=1'

## ----------------- ##
## Output variables. ##
## ----------------- ##

ACLOCAL='${SHELL} '\''/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/missing'\'' aclocal-1.16'
AMDEPBACKSLASH='\'
AMDEP_FALSE='#'
AMDEP_TRUE=''
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
AM_DEFAULT_VERBOSITY='1'
AM_V='$(V)'
AR='ar'
AUTOCONF='${SHELL} '\''/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/missing'\'' autoconf'
AUTOHEADER='${SHELL} '\''/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/missing'\'' autoheader'
AUTOMAKE='${SHELL} '\''/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/missing'\'' automake-1.16'
AWK='awk'
B2_LIBRARY_VERSION='1:4:0'
CC='gcc'
CCDEPMODE='depmode=gcc3'
CFLAGS='-O3'
CPPFLAGS='-DNDEBUG'
CYGPATH_W='echo'
DEFS='-DHAVE_CONFIG_H'
DEPDIR='.deps'
DLLTOOL='false'
DSYMUTIL='dsymutil'
DUMPBIN=':'
ECHO_C='\c'
ECHO_N=''
ECHO_T=''
EGREP='/usr/bin/grep -E'
EXEEXT=''
FGREP='/usr/bin/grep -F'
GREP='/usr/bin/grep'
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
LD='/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld'
LDFLAGS=' '
LIBOBJS=''
LIBS=''
LIBTOOL='$(SHELL) $(top_builddir)/libtool'
LIPO='lipo'
LN_S='ln -s'
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
MAKEINFO='${SHELL} '\''/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/missing'\'' makeinfo'
MANIFEST_TOOL=':'
MKDIR_P='./../src/6f75badf85-b3a5d32c8d.clean/install-sh -c -d'
NM='nm'
NMEDIT='nmedit'
OBJDUMP='objdump'
OBJEXT='o'
OPENMP_CFLAGS=''
OTOOL64=':'
OTOOL='otool'
PACKAGE='libb2'
PACKAGE_BUGREPORT='contact@blake2.net'
PACKAGE_NAME='libb2'
PACKAGE_STRING='libb2 0.98.1'
PACKAGE_TARNAME='libb2'
PACKAGE_URL='https://blake2.net'
PACKAGE_VERSION='0.98.1'
PATH_SEPARATOR=':'
RANLIB='ranlib'
SED='/usr/local/bin/gsed'
SET_MAKE=''
SHELL='/bin/sh'
SIMD_FLAGS=''
STRIP='strip'
USE_FAT_FALSE=''
USE_FAT_TRUE='#'
USE_SSE_FALSE=''
USE_SSE_TRUE='#'
VERSION='0.98.1'
ac_ct_AR='ar'
ac_ct_CC='gcc'
ac_ct_DUMPBIN='link -dump'
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE='#'
am__fastdepCC_FALSE='#'
am__fastdepCC_TRUE=''
am__include='include'
am__isrc=' -I$(srcdir)'
am__leading_dot='.'
am__nodep='_no'
am__quote=''
am__tar='$${TAR-tar} chof - "$$tardir"'
am__untar='$${TAR-tar} xf -'
bindir='${prefix}/tools/libb2/bin'
build='aarch64-apple-darwin20.4.0'
build_alias=''
build_cpu='aarch64'
build_os='darwin20.4.0'
build_vendor='apple'
datadir='${datarootdir}'
datarootdir='${prefix}/share/libb2'
docdir='${prefix}/share/libb2'
dvidir='${docdir}'
exec_prefix='${prefix}'
host='x86_64-apple-darwin'
host_alias='x86_64-apple-darwin'
host_cpu='x86_64'
host_os='darwin'
host_vendor='apple'
htmldir='${docdir}'
includedir='${prefix}/include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libb2/src/6f75badf85-b3a5d32c8d.clean/install-sh'
libdir='${exec_prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${prefix}/share/libb2'
mkdir_p='$(MKDIR_P)'
oldincludedir='/usr/include'
pdfdir='${docdir}'
pkgconfigdir='${libdir}/pkgconfig'
prefix='/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx'
program_transform_name='s,x,x,'
psdir='${docdir}'
runstatedir='${localstatedir}/run'
sbindir='${prefix}/tools/libb2/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "libb2"
#define PACKAGE_TARNAME "libb2"
#define PACKAGE_VERSION "0.98.1"
#define PACKAGE_STRING "libb2 0.98.1"
#define PACKAGE_BUGREPORT "contact@blake2.net"
#define PACKAGE_URL "https://blake2.net"
#define PACKAGE "libb2"
#define VERSION "0.98.1"
#define HAVE_MEMSET_S 1
#define HAVE_STDIO_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define HAVE_INTTYPES_H 1
#define HAVE_STDINT_H 1
#define HAVE_STRINGS_H 1
#define HAVE_SYS_STAT_H 1
#define HAVE_SYS_TYPES_H 1
#define HAVE_UNISTD_H 1
#define STDC_HEADERS 1
#define HAVE_STDDEF_H 1
#define HAVE_STDINT_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define NATIVE_LITTLE_ENDIAN 1
#define HAVE_MEMSET 1
#define HAVE_DLFCN_H 1
#define LT_OBJDIR ".libs/"

configure: exit 0

```
</details>

--- a/ports/libb2/portfile.cmake
+++ b/ports/libb2/portfile.cmake
@@ -16,7 +16,9 @@ endif()
 vcpkg_configure_make(
     AUTOCONFIG
     SOURCE_PATH ${SOURCE_PATH}
-    OPTIONS ${OPTIONS}
+    OPTIONS
+        ax_cv_check_cflags___O3=no # see https://github.com/microsoft/vcpkg/pull/17912#issuecomment-840514179
+        ${OPTIONS}
 )
 vcpkg_install_make()
 vcpkg_fixup_pkgconfig()

--- a/ports/libb2/vcpkg.json
+++ b/ports/libb2/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libb2",
-  "version-string": "0.98.1",
-  "port-version": 4,
+  "version": "0.98.1",
+  "port-version": 5,
   "description": "C library providing BLAKE2b, BLAKE2s, BLAKE2bp, BLAKE2sp",
   "homepage": "https://github.com/BLAKE2/libb2",
   "supports": "!windows"

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2974,7 +2974,7 @@
     },
     "libb2": {
       "baseline": "0.98.1",
-      "port-version": 4
+      "port-version": 5
     },
     "libbacktrace": {
       "baseline": "2021-03-14",

--- a/versions/l-/libb2.json
+++ b/versions/l-/libb2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0419d2f293a094dc4cdca1111daea2a6425a137b",
+      "version": "0.98.1",
+      "port-version": 5
+    },
+    {
       "git-tree": "98925560fbceb14da7820146f6df113f9dae7988",
       "version-string": "0.98.1",
       "port-version": 4

